libusb.go

```diff
@@ -188,7 +188,9 @@ func (libusbImpl) handleEvents(c *libusbContext, done <-chan struct{}) {
 		default:
 		}
 		if errno := C.libusb_handle_events_timeout_completed((*C.libusb_context)(c), &tv, nil); errno < 0 {
-			log.Printf("handle_events: error: %s", Error(errno))
+			if Error(errno) != ErrorInterrupted {
```

Comment:
Is that an issue related to this PR? With these changes patched manually in our code I can build for Windows just fine.

Reply:
Hi
> Is that an issue related to this PR?

I don't think so

Reply:
@zagrodzki is there anything we can do to help expediting getting this merged?

Reply:
> @zagrodzki is there anything we can do to help expediting getting this merged?

Look at this [gousb-action#build-and-test](https://github.com/google/gousb/actions/runs/7080912250/job/19279385474).
I think we should do something to make this [github action](https://github.com/google/gousb/blob/master/.github/workflows/build-and-test.yaml) go through.
It could be a code change, it could be a fix to the github workflow...

Reply:
I found others that have similar issues where most resolutions are upgrading the go version:
https://github.com/golang/go/issues/51007